io_scene_godot/converters/simple_nodes.py

```diff
@@ -72,6 +72,16 @@ def export_camera_node(escn_file, export_settings, node, parent_gd_node):
     return cam_node
 
 
+def emission_input(node_tree, name):
+    """Get the input from the emission shader node with the given name."""
+    for node in node_tree.nodes:
+        if node.bl_idname == 'ShaderNodeEmission':
+            for input in n.inputs:
+                if input.name == name:
+                    return input.default_value
+    return None
```

Comment:
Better to return some default value like `100` and throw a warning, otherwise a node tree with no `ShaderNodeEmission` would crash the program